modules/LoRA.py

```diff
@@ -25,7 +38,11 @@ def add_lora_to_model(lora_name):
             elif shared.args.load_in_8bit:
                 params['device_map'] = {'': 0}
 
-        shared.model = PeftModel.from_pretrained(shared.model, Path(f"{shared.args.lora_dir}/{lora_name}"), **params)
+        shared.model = PeftModel.from_pretrained(shared.model, Path(f"{shared.args.lora_dir}/{lora_names[0]}"), **params)
```

Comment:
Also related to the comment above, if the model is "fresh", is it necessary to reload it with `PeftModel.from_pretrained`?

Reply:
If I'm not mistaken, this isn't actually a full reload, this is just taking the non-peft model and wrapping it (+ applying the first lora). It definitely runs a lot faster than a full model load, and doesn't output any of the loading nonsense to console at least.